setup.py

```diff
@@ -125,23 +126,30 @@ def detect_gmp(self):
                 };
             '''.encode('ascii'))
             srcs = [temp.name]
+            trash.append(temp.name)
         exe = "a.out"
         try:
             objs = cc.compile(srcs)
+            map(trash.append, objs)
             cc.link(ccompiler.CCompiler.EXECUTABLE,
                     objs, exe,
                     libraries=['gmp', 'gmpxx'])
+            trash.append(exe)
         except Exception:
             # failure: remove the gmp dependency
             print('Failed to compile GMP source, disabling long support.')
-            for cfg in glob.glob(os.path.join(pythrandir, "pythran-*.cfg")):
-                with open(cfg, "r+") as cfg:
-                    content = cfg.read()
-                    content = content.replace('USE_GMP', '')
-                    content = content.replace('gmp gmpxx', '')
-                    cfg.seek(0)
-                    cfg.write(content)
-        map(os.remove, objs + srcs + [exe])
+            for pythrandir in (os.path.join(self.build_lib, "pythran"),
+                               os.path.join(os.path.dirname(__file__),
+                                            "pythran")):
```

Comment:
The configs in `self.build_lib` are staged for install while the one in `.` (the one updated in #515) are the ones that are loaded when you run `python setup.py test`. I think around here, you want to update both (even it is kind of a bad idea to update a version controlled file…).